cypress/utils/subscriptionModal.ts

```diff
@@ -0,0 +1,139 @@
+import { mockErrorResponse } from "./mockErrorResponse";
+
+export const openSubscriptionModal = (
+  route: string,
+  dataCyToggleModalButton: string
+) => {
+  cy.visit(route);
+  cy.dataCy(dataCyToggleModalButton).click();
+  cy.dataTestId("when-select").click();
+};
+
+export const testSharedSubscriptionModalFunctionality = (
```

Comment:
this is not really a util, it's a test, and should probably go in a test file that represents shared tests between the two modals